src/OrbitGl/TrackManagerTest.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_TRACK_MANAGER_TEST_H_
+#define ORBIT_GL_TRACK_MANAGER_TEST_H_
```

Comment:
This file only contains `struct TrackTestData`, so its name and the include guards should be updated. There's no reason for this file to match `TrackManagerTest.cpp`, in fact it probably shouldn't.

Reply:
Good point. I've moved this to a separate header and .cpp file.